herddb-core/src/main/java/herddb/model/ColumnTypes.java

```diff
@@ -34,8 +34,11 @@
     public static final int NULL = 5;
     public static final int DOUBLE = 6;
     public static final int BOOLEAN = 7;
+    public static final int NOTNULL_STRING = 9;
+    public static final int NOTNULL_INTEGER = 10;
+    public static final int NOTNULL_LONG = 11;
 
-    public static final int ANYTYPE = 10;
+    public static final int ANYTYPE = 12;
```

Comment:
Do not change values of constants, java compiler eagerly copies values to .class files, we will break compatibility.

Reply:
Agreed. I changed it back and moved the new types after value 10.